tests/feature_tests/requirements/requirements-dev.txt

```diff
@@ -701,44 +711,6 @@ pycodestyle==2.10.0 \
     --hash=sha256:347187bdb476329d98f695c213d7295a846d1152ff4fe9bacb8a9590b8ee7053 \
     --hash=sha256:8a4eaf0d0495c7395bdab3589ac2db602797d76207242c17d470186815706610
     # via flake8
-pydantic==1.10.7 \
```

Comment:
I saw it is indeed not in the `pyproject.toml`, did we unnecessarily include it before?

Reply:
Apparently :-)